From: Jakub Jirutka <jakub@jirutka.cz>
Date: Fri, 11 Mar 2022 23:56:33 +0100
Subject: [PATCH] Don't call pkexec directly

The goal is to allow users to use a method other than pkexec.

--- a/showmethekey-gtk/smtk-keys-emitter.c
+++ b/showmethekey-gtk/smtk-keys-emitter.c
@@ -454,7 +454,7 @@
 			G_SUBPROCESS_FLAGS_STDIN_PIPE |
 				G_SUBPROCESS_FLAGS_STDOUT_PIPE |
 				G_SUBPROCESS_FLAGS_STDERR_PIPE,
-			error, PKEXEC_PATH, PACKAGE_BINDIR "/showmethekey-cli",
+			error, PACKAGE_BINDIR "/showmethekey-cli-exec",
 			NULL);
 	}
 	// emitter->error is already set, just return.
